extensions/2.0/Vendor/EXT_feature_metadata/schema/featureCollection.property.schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Property array",
+  "type": "object",
+  "description": "Array containing property data",
+  "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+  "properties": {
+      "elementCount": {
+          "type": "integer",
+          "minimum": 1,
+          "description": "The number of elements in the array. `elementCount` may be greater than `featureCount` if the array contains extra elements not used by this model, e.g. if different models refer to different sections of a global array or if the arrays represents a larger data dictionary."
+      },
+      "elementByteLength": {
```

Comment:
:open_mouth: I didn't see this property before. I'll need to double check the featureClass definitions to make sure the wording doesn't contradict fixed-length strings/binary

Reply:
@lilleyse actually wait, how is this handled with UTF-8 encoding? suppose you set the byte length to 4 but put in the string

`|👀|` -- which is `\x7c\xf0\x9f\x91\x80\x7c` in binary (since emoji are far enough into the Unicode table that they take 4 bytes). 

If you chop it off after 4 bytes you get `\x7c\xf0\x9f\x91` you'll get an error about unexpected end of input.

Might be good to research how databases handle this case.

Reply:
Seems like a data problem not a spec problem? Each element would need to be a valid UTF-8 string, and in this case it is not.

Reply:
Yeah. I'll review your changes to make sure the wording makes sense.